frontend/src/components/elements/Buttons/ButtonLink/Button.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { StyledButton } from './Button.styles';
 
-export const ButtonLink = ({ children, props, onClick }) => (
-  <StyledButton onClick={onClick} {...props} type="button">
+export const ButtonLink = ({ children, props, onClick, borderActive }) => (
```

Comment:
Usuń props - komponent powinien być jak najmniej ingerowalny. Użytkownik komponentu powinien móc ingerować tylko w to co chcesz, i nic więcej.
To samo z drugim Buttonem.